pages/guides/agents/intermediate/langchain-rag-agent.mdx

```diff
@@ -5,9 +5,9 @@ import PackageVersion from 'components/package-version'
 
 ## Introduction
 
-In this guide, we'll walk through how to create agents capable of answering questions based on any provided document using the Fetch.ai `uagents` and `ai_engine` libraries as well as `openai` and `cohere`. The aim is to assists you in building a **LangChain Retrieval-Augmented Generation (RAG) Agent**!
+In this guide, we'll walk through how to create agents capable of answering questions based on any provided document using the Fetch.ai uAgents and ai_engine python libraries as well as openai and cohere. The aim is to assists you in building a **LangChain Retrieval-Augmented Generation (RAG) Agent**!
 
-The process of creating RAG agents has been improved by offering a decentralized and modular framework through the uagents and ai_engine libraries. These tools streamline the integration of AI models like OpenAI and Cohere, enabling more efficient and scalable development of intelligent agents. With enhanced interoperability, security, and resource management, this framework allows developers to quickly build and deploy sophisticated agents that can effectively answer questions based on any provided document, making the entire process faster and more robust.
+The process of creating RAG agents has been improved by offering a decentralized and modular framework through the uAgents and ai_engine libraries. These tools streamline the integration of AI models like OpenAI and Cohere, enabling more efficient and scalable development of intelligent agents. With enhanced interoperability, security, and resource management, this framework allows developers to quickly build and deploy sophisticated agents that can effectively answer questions based on any provided document, making the entire process faster and more robust.
```

Comment:
Is it uAgents? Shouldn't we keep everything Agent?

Reply:
`uagents` would be the package right? Then, Agents is for the rest?

Reply:
We never refer to ourselves as fetch, unless in conversation verbally, therefore we are Fetch.ai
uAgents should be cased, unless it's to describe code as in `import uagents` etc.